frontend/src/page/Login.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { signInWithPopup } from 'firebase/auth';
+import './login.css';
+import googlelogo from '../assets/google_logo.svg';
+import { auth, provider } from '../services/firebase';
+import { getUser } from '../api/user';
+import Form from '../components/Form';
+
+function Login() {
+  const [userName, setUserName] = useState('');
+  const navigate = useNavigate();
+  const [username, usernameToggle] = useState(false);
```

Comment:
What's this toggle for? I don't quite follow the logic in the Form component for it